test/dummy/config/environments/test.rb

```diff
@@ -29,6 +29,9 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
+  # run tests in a sorted order (and get rid of rails its lengthy deprecation warning)
+  config.active_support.test_order = :sorted
```

Comment:
Currently (Rails 4 and down) it's `:sorted`, in Rails 5 it will be `:random`, which is fine too – I just explicitly specified the value it already had implicitly to get rid of the deprecation-warning that was obfuscating the test-output. I guess it's for you to decide which you prefer. This is just what it already was.